OracleTuxedo/OracleTSAM/README.md

```diff
@@ -0,0 +1,174 @@
+* `TSAM_CONSOLE_ADMIN_PASSWD` The password of TSAM Manager console `admin` user. On successful initializing of the TSAM container, the TSAM console should be available at `http://<container_ip_address>:<wls_adm_svr_port>/tsam`. On the TSAM console login page, user name `admin` and this password will be used as the login credential.
+* `DBA_USER` The Oracle Database DBA username, typically it's `sys`.
+* `DBA_PASSWD` Password of database `sys` user.
+* `DB_TSAM_TBLSPACE` The tablespace in which the new TSAM database user will be created. For testing purpose, it would be OK to just use the `users` tablespace.
+* `WLS_PW` The admin user password of new created Oracle WebLogic domain.
+
+To start the container, simply run below command in the same directly where above `docker-compose.yml` file resides.
+
+```bash
+docker-compose up
+```
+
+> When running the container in a **Oracle Container Cloud Service** instance, just paste above `docker-compose.yml` file content into the **YAML** tab of the **Service Editor** on creating a container service.
+
+Besides the above environment variables which are required, there are also some optional ones could be used to provide more control on the container.
+
+* `DEBUG_MODE` Whether or not to turn on debug mode. The value is either `true` or `false` (default). When the debug mode is turned on, more debug messages will be printed out to the container stdout. For example, the passed in environment variables and corresponding values, the Oracle WebLogic domain startup log, etc. Furthermore, when debug is on, the container will never terminate even though error is occurred during the container initialization. While in the normal mode, the container terminates after 120 seconds on failure.
+* `DB_ENABLE_PARTITION` Whether or not to enable the Oracle Database Partitioning feature. The value is either `yes` or `no` (default). Please note only when a Oracle Database Enterprise Edition is being used, the database partitioning could be turned on. So for a Standard Edition or XE database, the value could only be `no`.
+* `ADMIN_PORT` The new created WebLogic domain Admin Server HTTP listening port, default is `7001`.
+* `ADMIN_SSL_PORT` The new created WebLogic domain Admin Server HTTPS listening port, default is `7002`.
+* `WLS_USER` The new created WebLogic domain admin user name, default is `weblogic`.
+* `DOMAIN_NAME` The new created WebLogic domain name, default is `tsamdomain`.
+
+### Running Oracle TSAM Plus with an existing TSAM database schema
+When an existing TSAM database is used for the TSAM Manager application, there will be no need to provide Oracle Database `sys` user related credentials. The `docker-compose.yml` looks like below:
+
+```yaml
+version: "2"
+services:
+  tsam:
+    image: oracle/tsam:12.2.2.1
+    hostname: tsam.docker
+    ports:
+      - 7001/tcp
+      - 22/tcp
+    privileged: true
+    environment:
+      - "DB_CONNSTR=db.box:1521/orcl"
```

Comment:
Here the `docker-compose` snippet is provided as a general sample, which is not differentiating the possible Oracle Database types, which could be XE/SE/CDB/PDB.